cockroachcloud/managing-cmek.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Managing Customer-Managed Encryption Keys (CMEK) for CockroachDB Dedicated
+summary: Tutorial for getting a dedicated cluster up and running with Customer-Managed Encryption Keys (CMEK)
+toc: true
+docs_area: manage.security
+---
+
+Customer-Managed Encryption Keys (CMEK) for {{ site.data.products.dedicated }} allows the customer to delegate responsibility for the work of encrypting their cluster data to {{ site.data.products.dedicated }}, while maintaining the ability to completely revoke {{ site.data.products.dedicated }}'s access.
+
+This page guides the user through the process of enabling Customer-Managed Encryption Keys (CMEK) for your {{ site.data.products.dedicated }} cluster.
+
+To follow this procedure requires admin access to your {{ site.data.products.dedicated }} organization, and the ability to create and manage identity and access management (IAM) and key management (KMS) services in your organization's cloud, i.e., your Google Cloud Platform (GCP) project or Amazon Web Services (AWS) account.
+
+See also:
+
+- [Customer Managed Encryption Key (CMEK) frequently asked questions (FAQ)](cmek-faq.html)
+- [Encryption at Rest (enterprise)](../{{site.versions["stable"]}}/security-reference/encryption.html#encryption-at-rest-enterprise)
+
+## Overview of CMEK management procedures
+
+This section gives a high level overview of the operations involved with implementing CMEK with your {{ site.data.products.dedicated }} cluster.
+
+- [Enabling CMEK](#enable-cmek) for a {{ site.data.products.dedicated }} requires several steps:
+
+	- Turning on the {{ site.data.products.dedicated }} CMEK feature.
+	- Provisioning an encryption key with your cloud provider's key management service (KMS).
+	- Granting {{ site.data.products.db }} access to the CMEK i.e., the key you control.
+	- Switching your cluster to use the CMEK.
+
+- [Checking CMEK status](#check-cmek-status) allows you to inspect the CMEK state of your cluster with a call to the Cluster API.
+
+- [Revoking CMEK for a cluster](#revoking-cmek-for-a-cluster) by revoking {{ site.data.products.dedicated }}'s access to your CMEK at the IAM/KMS level.
+
+- [Restore CMEK following a revocation event](#restore-cmek-following-a-revocation-event) by reauthorizing {{ site.data.products.db }} to use your key, and coordinating with our support team to assist in recovering your Organization.
+
+## Enable CMEK
+
+### Step 1. Prepare your {{ site.data.products.dedicated }} Organization
```

Comment:
At the top here you have a period after "Step" and the step number. Lower down you have a colon. Looking over other docs, looks as though we use periods after steps more widely. So, update the lower ones? I just realized I might have put ":" in some earlier feedback — sorry about that, not sure why that was in my head. So I would switch to period.